sdk/personalizer/Azure.AI.Personalizer/src/Models/PersonalizerClient.cs

```diff
@@ -57,11 +60,18 @@ public PersonalizerClient(Uri endpoint, TokenCredential credential, Personalizer
         /// <param name="endpoint"> Supported Cognitive Services endpoint. </param>
         /// <param name="credential"> A credential used to authenticate to an Azure Service. </param>
         /// <param name="isLocalInference"> A flag to determine whether to use local inference. </param>
+        /// <param name="configuration"> A configuration to use local reference. </param>
         /// <param name="options"> The options for configuring the client. </param>
-        public PersonalizerClient(Uri endpoint, TokenCredential credential, bool isLocalInference, PersonalizerClientOptions options = null) :
+        public PersonalizerClient(Uri endpoint, TokenCredential credential, bool isLocalInference, Configuration configuration, PersonalizerClientOptions options = null) :
```

Comment:
[nit] create constructors without default arguments as well

Reply:
`isLocalInference` needs a default value of `false` and `Configuration` should not be a parameter.

Reply:
[Why do we need to define another constructor without a default argument? Would the current one cover both definitions?] -- the SDK team asked me to add constructors without the default arguments as part of my sdk review. They explained that it is a better experience for users to have a constructor without default arguments. The additional default argument tends to cause confusion if it is not used.